gitops-deployment/README.md

```diff
@@ -18,6 +18,7 @@ For more information about ArgoCD, see the [ArgoCD documentation](https://argo-c
   - [Sync Applications](#sync-applications)
   - [Verify your Installation](#verify-your-installation)
 - [Customised Installation](#customized-installation)
+- [Upgrading IBM Cloudpak Foundational Services](#upgrading-ibm-cloudpak-foundational-services)
```

Comment:
Also note that when you change https://github.com/IBM/ibm-websphere-automation/pull/26/files#r2337409345, you will have to change this ToC entry as well.